src/runtime_src/core/pcie/linux/device_linux.cpp

```diff
@@ -128,6 +129,120 @@ struct bdf
   }
 };
 
+struct sdm_sensor_info
+{
+  using result_type = query::sdm_sensor_info::result_type;
+  using data_type = query::sdm_sensor_info::data_type;
+
+  static result_type
+  get(const xrt_core::device* device, key_type, const boost::any& reqType)
+  {
```

Comment:
please add some comments for future developer :)

Reply:
Please add comments.